opapi/test/api-types.test.ts

```diff
@@ -0,0 +1,23 @@
+import { describe, it } from 'vitest'
+import { createApi } from './api'
+import { join } from 'path'
+import { getFiles } from '../src/file'
+import { validateTypescriptFile } from './util'
+
+describe('api types generator', () => {
+  it('should be able to generate valid typescript files for section wise types', async () => {
+    const genClientFolder = join(__dirname, 'gen/api-types')
+
+    const api = createApi()
```

Comment:
**nitpick:** I know this one already exists, but I'd probably rename it `createDummyApi()` so its clear this is dummy data